training-job/bert.slurm

```diff
@@ -33,6 +33,10 @@ export LD_LIBRARY_PATH=/usr/local/lib/:$LD_LIBRARY_PATH
 # set the network interface
 export NCCL_SOCKET_IFNAME="en,eth,em,bond"
 
+
+# Uncomment for FSDP
+# ENABLE_NCCL_BASE_COLLECTIVES=0
```

Comment:
Please add the same check for the Mnist FSDP example

Reply:
added in PR #28